### ISSUE TITLE
False warning of FAA0001 when using 2D array

### ISSUE THREAD

**Issue by scott-moxham** [fluentassertions/fluentassertions.analyzers#309]
### Description

When testing the length of a 2D array is 0, the analyzer reports that I should use .HaveCount(0) but this isn't available for 2D arrays.

### Complete minimal example reproducing the issue

```csharp
// Arrange
var array = new string[0, 0];

// Assert
array.Length.Should().Be(0);
```

### Expected behavior:

Shouldn't be any issues flagged

### Actual behavior:

I get an FAA0001 issue flagged with the message "Use .Should().HaveCount()"

### Versions

* Which version of Fluent Assertions Analyzers are you using? 0.30.0
* Which .NET runtime and version are you targeting? .Net 8

**Comment by Meir017** [comment]
@scott-moxham thanks for reporting this! fix coming up

**Comment by scott-moxham** [comment]
Thanks @Meir017 for your super-fast response
